core/data_array.cpp

```diff
@@ -307,4 +307,16 @@ DataArray operator/(const VariableConstProxy &a, const DataConstProxy &b) {
   return DataArray(a / b.data(), b.coords(), b.labels(), b.masks(), b.attrs());
 }
 
+DataArray reciprocal(const DataConstProxy &a) {
+  return DataArray(reciprocal(a.data()), a.coords(), a.labels(), a.masks(),
+                   a.attrs());
+}
+
+DataProxy reciprocal(const DataConstProxy &a, const DataProxy &out) {
+  /* Coordinate checking is handled by reciprocal(VariableConstProxy,
+   * VariableProxy) */
```

Comment:
Variables do not have coordinates, how can this check anything?

Reply:
:man_facepalming: I was thinking of dimensions here.